components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/model/AuthenticatedIdPData.java

```diff
@@ -166,7 +168,38 @@ public boolean isAlreadyAuthenticatedUsing(String authenticatorName) {
             }
         }
 
-        if(log.isDebugEnabled()){
+        if (log.isDebugEnabled()) {
```

Comment:
Can't we just call isAlreadyAuthenticatedUsing(authenticatorName, authenticatorName) for this method?

Reply:
I think we can just keep this implementation, as there is no functionality change by doing the above.